lean4-infoview/src/infoview/messages.tsx

```diff
@@ -177,7 +177,8 @@ export function useMessagesForFile(uri: DocumentUri, line?: number): Interactive
             try {
                 const diags = await getInteractiveDiagnostics(rs, { uri, line: 0, character: 0 },
                     line ? { start: line, end: line + 1 } : undefined)
-                if (diags) {
+                if (diags && diags.length > 0) {
+                    // diags may be '' in some cases
```

Comment:
```suggestion
                    // diags may be [] when lake fails
```

There's also another one on line 101 for the "All Messages" section.

Reply:
Ok checking :)